tests/unit/__init__.py

```diff
@@ -10,7 +10,13 @@
 from __future__ import division
 from __future__ import absolute_import
 
+import os
 import sys
 
 # Add the generated thrift files for the unit tests.
 sys.path.append("build/thrift/v6/gen-py/")
+
+REPO_ROOT = os.path.abspath(os.environ['REPO_ROOT'])
```

Comment:
Why do we need this changes in the unit test `__init__`?

Reply:
Because we want to use the tu_collector as a lib from under the vendor directory. This can be done if its path is made available among the PYTHONPATH-ish places in line 27.